content/doc/book/managing/system-properties.adoc

```diff
@@ -79,7 +79,7 @@ Set to "true" to disable DNS multicast
 |hudson.FilePath.VALIDATE_ANT_FILE_MASK_BOUND |10000 |1.592 |
 Max. number of operations to validate a file mask (e.g. pattern to archive artifacts).
 
-|hudson.footerURL |http://jenkins-ci.org/ |1.416 |
+|hudson.footerURL |https://jenkins.io |1.416 |
```

Comment:
When we created this page originally, we should have escaped this URL, since it is not intended to be clickable in this context.
```suggestion
|hudson.footerURL |\https://jenkins.io |1.416 |
```